File: Examples/test-suite/ocaml/memberin_extend_runme.ml
open Swig
open Memberin_extend

let _ =
  let em1 = new_ExtendMe '() and em2 = new_ExtendMe '() in
  assert (em1 -> "[thing]" ("em1thing") = C_void);
  assert (em2 -> "[thing]" ("em2thing") = C_void);
  assert (em1 -> "[thing]" () as string = "em1thing");
  assert (em2 -> "[thing]" () as string = "em2thing");
;;

File: Examples/test-suite/lua/memberin_extend_runme.lua
local v=require("memberin_extend")

local em1 = v.ExtendMe()
local em2 = v.ExtendMe()
em1.thing = "em1thing"
em2.thing = "em2thing"
assert(em1.thing == "em1thing", "wrong: " .. em1.thing)
assert(em2.thing == "em2thing", "wrong: " .. em2.thing)
